deploy/helm/sumologic/values.yaml

```diff
@@ -4267,40 +4267,21 @@ falco:
     repository: sumologic/falco
     # pullSecrets: []
 
-  ## Add kernel-devel package through MachineConfig, required to enable building of missing falco modules (only for OpenShift)
-  addKernelDevel: true
-  ## Add initContainers to Falco pod
-  extraInitContainers:
-    ## Add initContainer to wait until kernel-devel is installed on host
-    - name: init-falco
-      image: public.ecr.aws/docker/library/busybox
-      command:
-        - 'sh'
-        - '-c'
-        - |
-          while [ -f /host/etc/redhat-release ] && [ -z "$(ls /host/usr/src/kernels)" ] ; do
-          echo "waiting for kernel headers to be installed"
-          sleep 3
-          done
-      volumeMounts:
-        - mountPath: /host/usr
-          name: usr-fs
-          readOnly: true
-        - mountPath: /host/etc
-          name: etc-fs
-          readOnly: true
```

Comment:
Why is this not needed anymore?

Reply:
Seems like falco architecture changed a little bit. I wanted to see what e2e tests will show. This configuration works fine for vagrant environment.

No I see that this is likely for openshift only. I will perform manual test then